src/ocean/net/device/Berkeley.d

```diff
@@ -1353,7 +1355,14 @@ deprecated public class IPv4Address : Address
         static uint parse(istring addr)
         {
                 char[64] tmp;
-                return ntohl(inet_addr(toStringz(addr, tmp)));
+                if ((addr.length + 1) > tmp.length)
```

Comment:
Is there really not a proper replacement of `toStringz` that checks for these kinds of conditions? I don't really like moving logic like this out of a helper function into the call site, just for the sake of deprecating the helper function.